BO4E/ENUM/Anfragetyp.cs

```diff
@@ -0,0 +1,66 @@
+using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Typ/Art der Anfrage (ORDERS ORDRSP IMD 7081)
```

Comment:
```suggestion
    /// Typ/Art der Anfrage (ORDERS/ORDRSP/QUOTES IMD 7081)
```

Reply:
hm ja in der quotes wird das aber bisher nicht verwendet, aber ja, technically schon